code/modules/mining/ore.dm

```diff
@@ -159,13 +159,13 @@
 		if (..())
 			return
 		for (var/mob/living/L in range(1,AST))
-			L.changeStatus("radiation", 5 SECONDS, 2)
+			L.take_radiation_dose(0.5 SIEVERTS)
 
 	onExcavate(var/turf/simulated/wall/auto/asteroid/AST)
 		if (..())
 			return
 		for (var/mob/living/L in range(1,AST))
-			L.changeStatus("radiation", 10 SECONDS, 2)
+			L.take_radiation_dose(1.0 SIEVERTS)
```

Comment:
in a full radsuit, this works out to be about 0.1Sv per rock mined, not including anything from the ores on the floor
this will probably work up to dangerous levels pretty quickly when mining cerenkite in bulk, even with "100%" protection.
maybe tone this down